asmcli/lib/util.sh

```diff
@@ -117,6 +117,15 @@ info() {
   fi
 }
 
+validation_error() {
+  error "${1}"
+  if only_validate; then
+    context_set-option "VALIDATION_ERROR" 1
```

Comment:
I think it would be nice if we could increment a counter here instead of just setting it to 1. That would let us print out the number of errors when we exit, and also make it easier to test.

Reply:
Done